x/nft/genesis.go

```
@@ -0,0 +1,23 @@
+package nft
+
+// ValidateGenesis check the given genesis state has no integrity issues
+func ValidateGenesis(data GenesisState) error {
+	for _, class := range data.Classes {
+		if err := ValidateClassID(class.Id); err != nil {
+			panic(err)
```

why panic instead of returning an error?